### ISSUE TITLE
The documentation link in the plugin takes me to GitHub rather than documentation

### ISSUE THREAD

**Issue by WayneCoombes** [bmarshall511/wordpress-zero-spam#287]
It is not harder than that.

**Comment by bmarshall511** [comment]
@WayneCoombes As expected, the documentation is in the Github Wiki (https://github.com/bmarshall511/wordpress-zero-spam/wiki). Are you looking for something specific?

**Comment by WayneCoombes** [comment]
Thanks for your reply. Q: Am I looking-for something specific? A: No. I was looking for USER documentation, first because I thought it was being offered, and second as an aid to things not clear to me on the settings panels… which is what I had expected to find: information to help fill in the blanks in my knowledge. I didn’t expect USER documentation to be on GitHub, and it wasn’t. When I looked in the Wiki on GitHub all I found were what I perceive to be command line instructions, not USER documentation. I’ll see if I can learn by doing.
Wayne
